packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1526,6 +1526,37 @@ export const GenerateContentTypesError = {
 	hint: (fileName?: string) =>
 		`This error is often caused by a syntax error inside your content, or your content configuration file. Check your ${fileName ?? 'content config'} file for typos.`,
 } satisfies ErrorData;
+/**
+ * @docs
+ * @message No parser found for file `users.csv`. Pass a parser to the `file` loader (e.g. `parser: csv`).
+ * @see
+ *  - [parsers](https://docs.astro.build/en/guides/imports/#parsers)
+ * @description
+ * Thrown when the `file` loader can’t determine which parser to use.
+ */
+export const FileParserNotFound = {
+	name: 'FileParserNotFound',
+	title: 'File parser not found',
+	message: (fileName: string) =>
+		`No parser was found for '${fileName}'. Pass a parser function (e.g. \`parser: mdx\`) to the \`file\` loader.`,
+} satisfies ErrorData;
+
+/**
+ * @docs
+ * @message Glob patterns are not supported in the `file` loader. Use the `glob` loader instead.
+ * @see
+ *  - [glob patterns](https://docs.astro.build/en/guides/imports/#glob-patterns)
```

Comment:
```suggestion
 *  - [Astro's built-in loaders](https://docs.astro.build/en/guides/content-collections/#built-in-loaders)
```
This is actually where we would send someone to learn about how they should properly use the file and glob loaders, so they can choose the right one, I think!

Reply:
Got, true.